src/Sulu/Bundle/PreviewBundle/Resources/js/containers/Preview/Preview.js

```diff
@@ -301,6 +305,24 @@ class Preview extends React.Component<Props> {
                                 options={this.availableDeviceOptions}
                                 value={this.selectedDeviceOption}
                             />
+                            <Toolbar.Popover
+                                icon="su-calendar"
+                                label={translate('sulu_admin.set_time')}
+                            >
+                                {() => (
+                                    <div className={previewStyles.dateTimeForm}>
```

Comment:
Did you implement a specific design here? Is it expected that there is a gap between the toolbar and the popover?
![Screenshot 2020-12-18 at 16 52 11](https://user-images.githubusercontent.com/13310795/102634018-8d143400-4151-11eb-85d2-5eb5cf9afa89.png)

Reply:
Are you using Firefox? This seems to work in Chrome, but not in Firefox... Will have a look.

Reply:
And I just realized, that there seems already to be an issue like that before this PR... The segments select also has a gap, although it is smaller than this one, but I guess it is for the same reason.
![Screenshot from 2020-12-21 09-11-29](https://user-images.githubusercontent.com/405874/102754122-c5438e80-436c-11eb-8e29-85d71e165f31.png)

Reply:
No I am using `Chrome 87.0.4280.88`

Reply:
Can you check if the segment dropdown also has a small gap in `master`? Because then it is related to the `PopoverPositioner` and not at all to this PR, would suggest to fix it in another PR then.

Reply:
no, i dont see a gap below the segment select 🙈 

![Screenshot 2020-12-22 at 09 40 00](https://user-images.githubusercontent.com/13310795/102867861-afe86600-4439-11eb-8994-ba6d1a0d4886.png)